app/controllers/catalog_controller.rb

```diff
@@ -409,11 +409,16 @@ def show
         field.include_in_advanced_search = false
         field.label = 'Browse by Call Number'
       end
-    end
 
-    config.add_search_field('browse_subjects') do |field|
-      field.include_in_advanced_search = false
-      field.label = 'Browse by Subject'
+      config.add_search_field('browse_authors') do |field|
```

Comment:
@awead + @banukutlu  - I moved all of the dropdown stuff inside `unless Settings.matomo_id.to_i == 7` so none of the browse dropdown options will display in prod. Let me know if you want me to revert this.